launchable/commands/subset.py

```diff
@@ -312,7 +314,11 @@ def run(self):
             if split:
                 click.echo("subset/{}".format(subset_id))
             else:
-                self.output_handler(output, rests)
+                _output, _rests = output, rests
```

Comment:
[question]
is it ok to switch output and rest even by default even if it is not `is_observation == true` ?

Reply:
OK, because we need to switch, if the `is_output_exclusion_rules` option is enabled 
https://github.com/launchableinc/cli/pull/435/files#diff-4c62e35fb4e3b2b6bff8593107cdddd1fd47eff90293edc9d838c9b2f3cff94aR339